### PR TITLE
Use DebugIds class

### PR DESCRIPTION
**Pull request by jclariviere**
I had to prefix the the constant names with `DBG` because it was conflicting with NameTokens in ui.xml files.

### CODE REVIEW

File: carstore/src/main/java/com/gwtplatform/carstore/client/application/ApplicationMobileView.ui.xml [hunk 1]
```diff
@@ -23,7 +25,7 @@
             <g:HTMLPanel ui:field="headerWrapper">
                 <g:SimplePanel ui:field="header" addStyleNames="{style.headerWrapper}"/>
 
-                <g:InlineLabel debugId="dom"/>
+                <g:InlineLabel debugId="{DBG_DOM}"/>
```
Comment (branflake2267) [inline carstore/src/main/java/com/gwtplatform/carstore/client/application/ApplicationMobileView.ui.xml]: Just something that came to mind when I saw this. Something that may be of interest, is using a GWT widget finder. Decouples using Ids to some extent. 
https://github.com/niloc132/gwt-driver/wiki

```
WebElement hlcElement = driver.findElement(
    new FasterByChained(By.xpath(".//div/*"), new ByWidget(driver,
        com.sencha.gxt.widget.core.client.container.HorizontalLayoutContainer.class)));
```

Comment (christiangoudreau) [inline carstore/src/main/java/com/gwtplatform/carstore/client/application/ApplicationMobileView.ui.xml]: Yeah, but's it's a lot faster to use debugId than "querying and finding" an element

File: carstore/src/main/java/com/gwtplatform/carstore/client/application/manufacturer/ManufacturerMobileView.ui.xml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
              xmlns:g='urn:import:com.google.gwt.user.client.ui'
              xmlns:cell="urn:import:com.google.gwt.user.cellview.client">
-    <g:ScrollPanel debugId="manufacturerPanel">
+
+    <g:ScrollPanel>
```
Comment (christiangoudreau) [inline carstore/src/main/java/com/gwtplatform/carstore/client/application/manufacturer/ManufacturerMobileView.ui.xml]: We should plan some time and review the whole design of this sample...

File: carstore/src/main/java/com/gwtplatform/carstore/client/debug/DebugIds.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package com.gwtplatform.carstore.client.debug;
```
Comment (christiangoudreau) [inline carstore/src/main/java/com/gwtplatform/carstore/client/debug/DebugIds.java]: license header

**Comment by christiangoudreau** [comment]
LGTM

File: carstore/src/main/java/com/gwtplatform/carstore/client/application/ApplicationMobileView.ui.xml (hunk 1 above)
Comment (branflake2267) [inline carstore/src/main/java/com/gwtplatform/carstore/client/application/ApplicationMobileView.ui.xml]: Good point :)

**Comment by meriouma** [comment]
LGTM

**Comment by christiangoudreau** [comment]
PING

**Pull request merged**